src/basedtyping/transformer.py

```diff
@@ -0,0 +1,253 @@
+"""utilities to create standard compatible annotations"""
+
+from __future__ import annotations
+
+import ast
+import types
+import typing
+import uuid
+from contextlib import contextmanager
+from enum import Enum
+from typing import cast
+
+import typing_extensions
+from typing_extensions import override
+
+import basedtyping
+
+
+# ruff: noqa: S101
+class CringeTransformer(ast.NodeTransformer):
```

Comment:
I believe the academic term for this is "desugaring" (although I the current name better)

Reply:
i do too

Reply:
i think "cringification" fits better, as "desugaring" implies it's a super set, but it's actually incompatible under certain configurations

Reply:
> i think "cringification" fits better, as "desugaring" implies it's a super set, but it's actually incompatible under certain configurations

Ah, I believe the academic term for that is a "basedset"